nginx/nginx-1.16.patch

```diff
@@ -50,7 +50,7 @@ index 2c7af1040..abf920bae 100644
 @@ -25,6 +25,10 @@ if [ $USE_OPENSSL = YES ]; then
      . auto/lib/openssl/conf
  fi
- 
+
```

Comment:
looks like most of nginx diff is not intended other than new API use?

Reply:
bah, editor stripping whitespace